config/keys.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   jwt:{
     secret_key: '8ht98h93tge4219h98hg98h38eahr9guhea' //DEFINE KEY
```

Comment:
wachtwoorden gaan voortaan in de .env file. wanneer je wachtwoorden tegenkomt (ook al heb je die er zelf niet in gezet, plaats deze dan in de env). tevens. wanneer de env file is aangepast. plaats de nieuwe op drive.